stacks/java-wildfly/2.0.2/devfile.yaml

```
@@ -0,0 +1,173 @@
+schemaVersion: 2.2.2
+metadata:
+  name: wildfly-start
+  version: 2.0.2
+  website: https://wildfly.org
+  displayName: WildFly Getting Started
+  description: Upstream WildFly Getting Started
+  icon: https://design.jboss.org/wildfly/logo/final/wildfly_logomark.svg
+  tags: ['Java', 'WildFly']
+  projectType: 'wildfly'
+  language: 'java'
+variables:
+  imageRegistry: 'quay.io'
+  imageRegistryNamespace: toBedefined
+  imageRegistryLogin: toBedefined
+  imageRegistryPassword: toBedefined
+  imageName: 'getting-started'
+  imageVersion: 'latest'
+  applicationName: 'start'
+  nodeName: 'getting-started'
+starterProjects:
+  - name: getting-started
+    description: WildFly Getting Started
+    git:
+      checkoutFrom:
+        remote: wildfly-devfile-examples
+        revision: getting-started-2.0.2
+      remotes:
+        wildfly-devfile-examples: https://github.com/wildfly-extras/wildfly-devfile-examples.git
+components:
+  - name: tools
+    container:
+      image:  quay.io/devfile/universal-developer-image:ubi8-latest
+      memoryLimit: 1512Mi
+      mountSources: true
+      volumeMounts:
+        - name: m2
+          path: /home/user/.m2
+      env:
+        - name: JAVA_OPTS
+          value: '-Djava.security.egd=file:/dev/urandom -Djboss.host.name=localhost'
+        - name: DEBUG_PORT
+          value: '5005'
+        - name: NODE_NAME
+          value: '{{nodeName}}'
+        - name: IMAGE_REGISTRY
+          value: '{{imageRegistry}}'
+        - name: IMAGE
+          value: '{{imageName}}'
+        - name: IMAGE_REGISTRY_NAMESPACE
+          value: '{{imageRegistryNamespace}}'
+        - name: IMAGE_REGISTRY_LOGIN
+          value: '{{imageRegistryLogin}}'
+        - name: IMAGE_REGISTRY_PASSWORD
+          value: '{{imageRegistryPassword}}'
+      endpoints:
+        - name: debug
+          exposure: internal
+          protocol: tcp
+          targetPort: 5005
+        - name: 'http'
+          protocol: http
```

```suggestion
          protocol: https
```

Could we change this?